### PR TITLE
Fixed error with searching 'Follow' and 'Following' buttons

### PR DESCRIPTION
**Pull request by dbknv**
I found issue when I'm using like by tag. For example:
`session.like_by_tags(['hashtag'], amount=1)`

The main problem was with "find_element_by_xpath", after my fix it starts work properly.

### CODE REVIEW

File: instapy/unfollow_util.py [hunk 1]
```diff
@@ -128,12 +128,9 @@ def follow_user(browser, follow_restrict, login, user_name):
     """Follows the user of the currently opened image"""
 
     follow_button = None
-    flwBtn = browser.find_elements_by_xpath("//article/header/span/button")
-    if len(flwBtn) == 0:
-        flwBtn = browser.find_elements_by_xpath('//*[@id="react-root"]/section/main/article/header/div[2]/div[1]/span/span[1]/button')
-        body_elem = browser.find_element_by_tag_name('body')
```
Comment (timgrossmann) [inline instapy/unfollow_util.py]: Why is this needed?

**Review by timgrossmann — CHANGES_REQUESTED** [review]

File: instapy/unfollow_util.py (hunk 1 above)
Comment (dbknv) [reply instapy/unfollow_util.py]: I don't exactly know what this code do. I suggest this is some kind searching 'Follow' button with reward to Home. When  I use this code script always gets error like this: 

> [1/2]
Image from: b'motofame'
Link: b'https://www.instagram.com/p/BYPUSudheIw/?tagged=gopro'
Description: b'Car or bike?
--> Image Liked!
--> Not commented
Traceback (most recent call last):
  File "C:/MyData/Instagram-Bot_Test.py", line 13, in <module>
    session.like_by_tags(['gopro'], amount=2)
  File "C:\MyData\\instapy\instapy.py", line 539, in like_by_tags
    followed += follow_user(self.browser, self.follow_restrict, self.username, user_name)
  File "C:\MyData\\instapy\unfollow_util.py", line 138, in follow_user
    follow_button = flwBtn[0]
IndexError: list index out of range

I can't explain why others don't get the same issue.

**Review by timgrossmann — APPROVED** [review]

**Comment by timgrossmann** [comment]
@sterid Thank you for the changes, I'll test it and merge it once it's working

Thank you for contributing

**Comment by balakedev** [comment]
@timgrossmann this seems to have worked for me(RE: #602 ), a bit of a band-aid fix at least. I have received this error once:
```
Image check error: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//img[@class = "_2di5p"]"}
  (Session info: chrome=60.0.3112.113)
  (Driver info: chromedriver=2.30.477700 (0057494ad8732195794a7b32078424f92a5fce41),platform=Windows NT 10.0.14393 x86_64)
```
 However, it seems to be working fine other than that one post! Would be interested another users opinion on the matter regardless though.

**Pull request merged**